docs/source/AdministratorGuide/Resources/Storages/index.rst

```diff
@@ -0,0 +1,242 @@
+.. _resourcesStorageElement:
+
+---------------
+StorageElements
+---------------
+
+
+
+DIRAC provides an abstraction of a SE interface that allows to access different kind of them with a single interface. The access to each kind of SE ( SRMv2, DIRAC SE, ...) is achieved by using specific plugin modules that provide a common interface. The information necessary to define the proper plugin module and to properly configure this plugin to access a certain SE has to be introduced in the DIRAC :ref:`Configuration <dirac-cs-structure>`. An example of such configuration is::
+
+    CERN-USER
+    {
+      ReadAccess = Active
+      WriteAccess = Active
+      AccessProtocol.1
+      {
+        PluginName = SRM2
+        Access = remote
+        Protocol = srm
+        Host = srm-lhcb.cern.ch
+        Port = 8443
+        WSUrl = /srm/managerv2?SFN=
+        Path = /castor/cern.ch/grid
+        SpaceToken = LHCb_USER
+      }
+    }
+
+
+
+Configuration options are:
+
+* `BackendType`: just used for information. No internal use at the moment
+* `SEType`: Can be `T0D1` or `T1D0` or `T1D1`. it is used to asses whether the SE is a tape SE or not. If the digit after `T` is `1`, then it is a tape.
+* `UseCatalogURL`: default `False`. If `True`, use the url stored in the catalog instead of regenerating it
+* `ChecksumType`: default `ADLER32`. NOT ACTIVE !
+* `Alias`: when set to the name of another storage element, it instanciates the other SE instead.
+* `ReadAccess`: default `True`. Allowed for Read if no RSS enabled
+* `WriteAccess`: default `True`. Allowed for Write if no RSS enabled
+* `CheckAccess`: default `True`. Allowed for Check if no RSS enabled
+* `RemoveAccess`: default `True`. Allowed for Remove if no RSS enabled
+
+
+
+-------------------
+StorageElementBases
+-------------------
+
+In order to factorize the configuration, it is possible to use `BaseSE`, which acts just like inheritance in object programming. You define a SE just like any other but in the `StorageElementBases` section. This SE can then be refered to by another SE. This new SE will inherit all the configuration from its parents, and can override it.  For example::
```

Comment:
I would start this sub-section with the following: "Installations tend to have several StorageElements, with very similar configurations (e.g., the same Host and Port). It could be useful to factorize the SEs configuration to avoid repeating it"